tox.ini

```diff
@@ -1,2 +1,3 @@
 [flake8]
-max-line-length = 120
```

Comment:
I know, but the request was to restore black functionality, and this was required for black and flake8 to be compatible. Enabling black without having it rewrite files will generate errors, which can only be addressed by fixing the line length issue.
I would rather have left black disabled for the GHA build, but the request was to add it back in. The fact that the flake8 config was not black compliant was a mistake, afaik.
I'd say this is also a reason to try harder to have services written in the same language at KBase be compatible in terms of tooling and practices. Otherwise, it is hard when entering a project to discover these sorts of issues until late in the lifecycle of a set of changes (PR).